pkg/jobrunaggregator/jobrunaggregatorapi/types_row_historical_data.go

```diff
@@ -33,7 +41,10 @@ type AlertHistoricalDataRow struct {
 	AlertName      string
 	AlertNamespace string
 	AlertLevel     string
+	FirstObserved  time.Time
```

Comment:
There were only needed for alerts so let's move there

Reply:
they are in alerts